rflx/templates/rflx_generic_types.ads

```diff
@@ -1,5 +1,7 @@
 pragma Style_Checks ("N3aAbcdefhiIklnOprStux");
 
+pragma Warnings (Off, "type ""Bytes_Ptr"" is not referenced");
+
```

Comment:
Why do we need to suppress this warning?

Reply:
This is needed, because `Bytes_Ptr` is not used in `Generic_Types` anymore, but `Generic_Types` still acts as a container for all user-defined types by the rest of the generated code. A user only has to create one instance of `Generic_Types` with his own `Bytes`-related types, which can than be used for instantiating message packages. If `Bytes_Ptr` is removed as generic parameter of `Generic_Types`, it would have to be added to all generic message packages (additionally to the already existing instance of `Generic_Types`), so that a user would always have to provide an instance of `Generic_Types` and a compatible `Bytes_Ptr` type to instantiate a message package.

Reply:
I clearly prefer to have on central types package.